final_race:
  ros__parameters:
    num_of_samples: 300
    look_ahead_dist: 1.75
    goal_sample_rate: 0.3
    max_yaw: 1.25
    max_vel: 6.0
    max_steering: 0.4236
    steering_gain: 0.5
    velocity_gain: 0.5
    time_step: 0.05
    extend_step: 5
    grid_width: 100
    grid_height: 150
    grid_resolution: 0.02
    safety_padding: 0.3305
    disparity_extend_range: 0.2032
    valid_path_length: 4
    waypoint_file_path: /sim_ws/src/final_race/waypoints/atrium_waypoints.txt
    waypoint_frame_id: map
    pose_to_listen: ego_racecar/base_link
    pose_topic: ego_racecar/odom